lib/loc.ml

```diff
@@ -11,7 +11,8 @@
 (* Locations management *)
 
 type source =
-  | InFile of string
+  (* OCaml won't allow using DirPath.t in InFile *)
```

Comment:
Why not?

Reply:
Perhaps childhood trauma?  I assumed it was a circular linkage issue and left it at that.  Saving it as a string is fine, too.